src/com/palmergames/bukkit/towny/TownyLogFormatter.java

```diff
@@ -15,7 +15,7 @@
 	@Override
 	public synchronized String format(LogRecord record) {
 
-		StringBuffer buf = new StringBuffer(180);
+		StringBuilder buf = new StringBuilder(180);

```

Comment:
Why are we moving to a non-thread-safe method?

Reply:
Afaik this doesn't matter for a method, it only matters if you have a class variable which is being accessed by multiple threads, however since this is one method, and it just creates the buffer each time the method is loaded, this shouldn't matter, however correct me if I'm wrong